src/python/.gitignore

```diff
@@ -0,0 +1 @@
+gen/
```

Comment:
I guess I was too vague with my nit. :-P I mean that in `commands.py` it appears that the folder is named `gens/` but here `gen/` is being ignored.